tests/www/views/test_views_log.py

```diff
@@ -461,7 +461,7 @@ def test_redirect_to_external_log_with_local_log_handler(log_admin_client, task_
     )
     response = log_admin_client.get(url)
     assert 302 == response.status_code
-    assert 'http://localhost/home' == response.headers['Location']
+    assert '/home' == response.headers['Location']
```

Comment:
Should this be considered breaking compatibility?

Reply:
I tihnk not - it will work in general - it will only redirect using path rather than full URL, I guess in any browser it will be treated properly and redirection will work as the browser will automatically pre-pend the host if it's the same as previously. 

I can't imagine scenario where it will create a problem (unless I am mistaken about the circumstances of this different Flask behaviour)